superset/reports/commands/exceptions.py

```diff
@@ -155,6 +155,10 @@ class AlertQueryError(CommandException):
     message = _("Alert found an error while executing a query.")
 
 
+class AlertQueryTimeout(CommandException):
+    message = _("A timeout occurred while executing the query.")
```

Comment:
As this can happen during either the query or the thumbnail computation, perhaps we should reword this as "A timeout occurred while executing the background job." - is this the same message that would trigger when a Report times out?

Reply:
not the same message, this is really specific for queries

Reply:
created a specific exception for screenshots timeout also